hnn_core/gui/_viz_manager.py

```diff
@@ -365,11 +365,11 @@ def _add_figure(b, widgets, data, scale=0.95):
     with fig_outputs:
         dpi = _dpi
         figsize = (scale * ((int(viz_output_layout.width[:-2]) - 10) / dpi),
-                scale * ((int(viz_output_layout.height[:-2]) - 10) / dpi))
+                   scale * ((int(viz_output_layout.height[:-2]) - 10) / dpi))
```

Comment:
if you typecast it to `int` does that mean `viz_output_layout.height[:-2]` gives a single element? Does that then mean you can just use a single index? I guess `viz_output_layout.height[0]` ?

Reply:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/11160442/187007151-3e9b02b8-b1d8-4f42-8465-3a607ba7e4a1.png">
I don't get it. Is this what you mean?

Reply:
ohhh ... I see, I understand now. Sorry about the noise